leaf-ui/rappid-extensions/AssignmentsTable.js

```diff
@@ -150,7 +150,7 @@ var AssignmentsTable = Backbone.View.extend({
     displayAbsoluteIntentionAssignments: function() {
         this.model.getIntentions().forEach(intentionCell => {
             var intentionBbm = intentionCell.get('intention');
-            var funcSegList = intentionBbm?.get('evolvingFunction')?.get('functionSegList');
+            var funcSegList = intentionBbm.get('evolvingFunction').get('functionSegList');
```

Comment:
If you remove the question marks I think we will just need a check to see if the evolvingFunction is false. That is probably better form though so good on removing question marks, but do check that evolvingFunction isn't null before trying to get the function segment list. Then, instead of checking if funcSegList is undefined check if its empty

Reply:
Ah ok, I thought those were typos